### ISSUE TITLE
Docs example for Popup

### ISSUE THREAD

**Issue by nickforddesign** [soal/vue-mapbox#86]
There is a minor bug in the example code for Popups:

https://soal.github.io/vue-mapbox/guide/markers&popups.html#popup

```html
<MglPopup :coordinates="coordinates" :anchor="top">
  <VCard> <div>Hello, I'm popup!</div> </VCard>
</MglPopup>
```

either should be `anchor="top"` or `:anchor="'top'"`, otherwise `top` must be a member of the component.

**Comment by nickforddesign** [comment]
Also, as pointed out in https://github.com/soal/vue-mapbox/issues/55, coordinates should not be passed by default, that was confusing

**Comment by soal** [comment]
Fixed in version 0.2
